src/context-selection/section-filter-selector-bar-item/use-data-set-sections-info.js

```diff
@@ -21,15 +21,7 @@ export default function useDataSetSectionsInfo() {
         data,
     } = useQuery(queryKey, { enabled: !!dataSetId })
 
-    const sections = data?.dataSet.sections.map(({ id, displayName }) => ({
-        value: id,
-        label: displayName,
-    }))
-
-    const dataSet = {
-        ...(data?.dataSet || {}),
-        sections,
-    }
+    const dataSet = data?.dataSet || null
```

Comment:
This would resolve to undefined if data is null/undefined. Do we need it to be null if data is `undefined`?

Reply:
You're absolutely right, I changed it